haiku-apps/belife/belife-1.0.0.recipe

```diff
@@ -14,7 +14,10 @@ HOMEPAGE="https://github.com/HaikuArchives/BeLife"
 COPYRIGHT="2006 Studio-33"
 LICENSE="BeLife License"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/96e31e8b4543723c7c8ea4a3d0c57c8cd52f82ca.tar.gz"
+srcGitRev="96e31e8b4543723c7c8ea4a3d0c57c8cd52f82ca"
+SOURCE_URI="${HOMEPAGE}/archive/${srcGitRev}.tar.gz"
```

Comment:
Yeah i've do this before, but i reverted it because that secondaryArchSuffix_devel give me a problem.